lib/reports/BackupReportManager.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+const moment = require('moment-timezone');
+const Repository = require('../db').Repository;
+const CONST = require('../constants');
+const config = require('../config');
+class BackupReportManager {
+  static createInstanceBackupRecord(instance_id, start_time, end_time) {
```

Comment:
can rename this to `computeInstanceBackupSummary` or `getInstanceBackupSummary` - as you are not creating anything in this method. Use the verbs to indicate what the methods are doing. `create` and `update` are strong verbs which should be indicative of some resources actually being created/updated.